src/hooks/__tests__/useIsValidExecution.test.ts

```diff
@@ -102,18 +115,22 @@ describe('useIsValidExecution', () => {
       },
     })
 
-    const { result } = renderHook(() => useIsValidExecution(createSafeTx(), BigNumber.from(1_000)))
+    const { result } = renderHook(() => useIsValidExecution(mockTx, mockGas))
 
-    const { isValidExecution, executionValidationError, isValidExecutionLoading } = result.current
+    var { isValidExecution, executionValidationError, isValidExecutionLoading } = result.current
```

Comment:
`var`? :)

Reply:
I was having reassignment issues and found [this example in the `useLoadChains` test](https://github.com/safe-global/web-core/blob/dev/src/hooks/__tests__/useLoadChains.test.ts#L39-L51) that helped.

A fresh head has helped though, I've adjusted it to use `let` instead.